fedimint-testing/src/bin/fixtures.rs

```diff
@@ -151,11 +151,13 @@ async fn await_fedimint_block_sync() -> anyhow::Result<()> {
 }
 
 async fn run_bitcoind() -> anyhow::Result<()> {
-    let current_dir = env::current_dir().unwrap();
+    let btc_dir = env::var("FM_BTC_DIR")?;
+    let current_dir = env::current_dir()?;
     let conf_path = current_dir.join("misc/test/bitcoin.conf");
-    let conf_path_string = conf_path.into_os_string().into_string().unwrap();
-    //Spawn bitcoind
+    let conf_path_string = conf_path.into_os_string().into_string()?;
```

Comment:
nit: use `Path::to_str()`

Reply:
can't I use  `let conf_path_string = conf_path.into_os_string().into_string().unwrap();`?

Reply:
you can, 
but the alternative is better
`let conf_path_string = conf_path.to_str().context("path must be valid UTF-8)?;`